Source/WebKit/ModelProcess/cocoa/ModelProcessModelPlayerProxy.mm

```diff
@@ -254,18 +254,20 @@ static ResourceError toResourceError(String payload, Model& model)
 
 bool ModelProcessModelPlayerProxy::transformSupported(const simd_float4x4& transform)
 {
+    static const float tolerance = 1e-5f;
```

Comment:
```suggestion
    static constexpr float tolerance = 1e-5f;
```

Reply:
No need for static either afaik with function local constexpr